modules/core/src/test/java/org/apache/ignite/internal/metric/CacheMetricsAddRemoveTest.java

```diff
@@ -204,12 +253,14 @@ private void checkMetricsEmpty(String cachePrefix) {
 
             assertNull(mreg.findMetric(metricName(cachePrefix, CACHE_GETS)));
             assertNull(mreg.findMetric(metricName(cachePrefix, CACHE_PUTS)));
+            assertNull(mreg.findMetric(metricName(cachePrefix, GET_TIME)));
```

Comment:
Here and above, we must use mreg.findMetric(<metric_name>) without registry prefix. Otherwise, this check makes no sense and always true.

Reply:
Thanks, fixed.